### ISSUE TITLE
Wrangler keeps crashing

### ISSUE THREAD

**Issue by rishavs** [cloudflare/wrangler#1995]
#### Describe the bug

Wrangler keeps crashing for me every 5-10 mins.
I do not see any error message of why it happened.

trying to send wrangler report also fails.
but wrangler whoami works fine and I can see my token and mail.

![image](https://user-images.githubusercontent.com/1210239/125678930-e2af0a43-84a1-4089-bb3e-993942a65e0b.png)

#### Environment and versions

Fill out the following information about your environment.

* operating system:
* output of `wrangler -V`: wrangler 1.17.0
* output of `node -v`: v16.4.2
* content of `wrangler.toml`:
```
name = "myapp"

type = 'webpack'
account_id = 'myaccountid'
route = ''
zone_id = ''
usage_model = ''
workers_dev = true
target_type = "webpack"
```

**Comment by jgentes** [comment]
@rishavs There was an issue with Wranger report - could you try again? We haven't addressed the underlying issue but the report should come through to us. Also we've released v1.18.

**Comment by Electroid** [comment]
Fixed in #2008, which is being released today!

**Comment by rishavs** [comment]
haven't had a chance to check wrangler but can confirm that the wrangler repot is working now.
